docs/_docs/Starters Academy (LDSSA)/Capstone-Project.md

```diff
@@ -0,0 +1,116 @@
+
+1. check if it's working
+2. if not, debug why
+3. check it's performance (how many requests you were able to answer, for example)
+
+This means that ideally you should have your app ready by then. This is not mandatory, as it is not an evaluation moment, but it is definitely recommended.
+
+
+#### App Test round
+
+Once you deliver your app and the first report, it's game on! We'll start sending requests into your app, in two rounds:
+
+* First, we'll send requests that you should be able to make predictions for 
+* Second, we'll send updates with the ground truth so that you can use it for future usage
+
+Don't worry if you don't understand right now what this will mean, you'll get there. All you need to know is that at this point you must deliver a working app, i. e., it should return some prediction for our data. 
+
+
+#### Stage 2
+
+In the second part of the project, you will analyse the real data you received and produce a second report going through it, retraining your model and redeploying it. 
+
+At the end of this you will produce a second report and will have a smaller app test round. Redeploying your model is not mandatory, but recommended to show improvements in your results.
+
+
+#### Comment rounds
+
+In each stage you produce a version of the report. This is an opportunity for you to receive feedback and improve your reports.
+
+* There is only 1 comment round per report
+* Information about pass/fail is not mentioned by instructors
+* Disambiguation should not be expected as a standard, i.e., you may ask a question in the comments that gets no answer, but can be provided by instructors if they deem it necessary
+* You can make changes that were not requested in the comments; however, instructors will be looking for things that they did mention;
+
+
+### Time commitment
+You should be prepared to spend a recommended *10 hours per week* on the course and an absolute minimum of *5 hours per week* in case you are having a busy week.
+
+
+
+### Capstone Evaluation Rules
+
+In order to pass the capstone all the following must be delivered and pass a set of standards:
+
+* Report 1
+* Working API
+* Report 2
+* All code (API + training + analysis)
+
+#### API
+
+All students should have delivered a working API by report 1 to pass. This will be verified by the following:
+
+* the API responds to the requests - anything other than a 404;
+* an intermediate delivery of the app code may be requested for confirmation
+
+
+#### Report
+
+In order to get a passing chance, both reports need to be sent with **all sections**. In addition, each section has to pass a minimum threshold of quality, with respect to the requirements presented. You can refer to the [sections and descriptions of batch 4](TBD) to understand better how these may look.
+
+#### Automatic fails
+
+The capstone will automatically be considered a fail if:
+
+* Empty section
+* Section with 1/2 sentences
```

Comment:
should this be "1-2" instead of "1/2", or did you mean half sentences ?

Reply:
Fixed